assets/js/hooks/local-storage.js

```diff
@@ -17,10 +17,8 @@ export const hooks = {
   restore(obj) {
     var authData = localStorage.getItem(obj.auth_key)
     var roomCount = localStorage.getItem(obj.room_count_key)
-   
-    if (authData && roomCount) {
-      this.pushEvent(obj.event, {auth: authData, room_count: Number(roomCount)})
-    }
+    var responseData = authData && roomCount ? {auth: authData, room_count: Number(roomCount)} : {}
```

Comment:
`roomCount` might be absent. Will this make current localstorage to be owerwritten?

Reply:
Yes, this will do this in the `storeRoom` handler: 
````
> Number(null)
0
````

Reply:
Also I change the condition for `responseData`. I mean that there is no problem when `roomCount` is absent and then we push it to server as `0`
````
> Number(null)
0
````